lib/posix/Kconfig

```diff
@@ -4,6 +4,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+config	POSIX_MAX_FDS
+	int "Maximum number of open file descriptors"
+	default 16 if POSIX_API
+	default 4
```

Comment:
There is relation of this value and for example CONFIG_NET_MAX_CONTEXTS value. I wonder if we could tie them together somehow.

Reply:
> There is relation of this value and for example CONFIG_NET_MAX_CONTEXTS value. I wonder if we could tie them together somehow.

Just the same as on POSIX_MAX_OPEN_FILES below. Tie - no. Better document/cross-reference this relations - yes, over time.

Reply:
Sure, I was not really suggesting we start to tweak this now, just pondering this issue and how to solve it.